ovirtlago/virt.py

```diff
@@ -300,18 +321,12 @@ def _host_is_up():
                     )
                 elif host_obj.status == sdk4.types.HostStatus.INSTALL_FAILED:
                     raise RuntimeError('Host %s installation failed' % h.name)
-                elif host_obj.status == sdk4.types.HostStatus.NON_RESPONSIVE:
```

Comment:
Because it failed the run, but after a few seconds the host reported 'UP' in the engine.